packages/search/package.json

```diff
@@ -41,9 +41,12 @@
     "search"
   ],
   "dependencies": {
-    "@lion/combobox": "^0.5.1",
-    "@open-wc/scoped-elements": "^1.3.2",
+    "@lion/combobox": "^0.8.0",
+    "@lion/core": "^0.18.0",
+    "@lion/listbox": "^0.10.1",
+    "@open-wc/scoped-elements": "^2.0.0-next.3",
     "chalk": "^4.0.0",
+    "lit": "^2.0.0-rc.2",
```

Comment:
that one is not needed then right?

Reply:
ah, not anymore since we depend o @lion/core. Yep, will change